runner/ansible/README.md

```diff
@@ -22,3 +27,38 @@ These are the fields needed by Trento:
 is that the labels are used for control purpose (select all the checks with this label e.g.), and the groups are used just for visual purposes
 - `description`: A longer description about the test purpose. It can be written using markdown.
 - `implementation`: Usually the task `main.yml` content
+
+## Check files
+
+The check files contain the actual task which the runner will execute. An
+example is available at [tasks/main.yml](roles/checks/1.1.1/tasks/main.yml).
+These files are written like a normal ansible task.
+
+## Workflow
```

Comment:
Let's remove completely this `Workflow` page. I don't want to explain here how to setup the runner.
This document only aims to the Check implementation itself

Reply:
It was not meant to explain the setup but to tell that a restart of the services are needed. Should I shorten it or remove it?

Reply:
I would totally remove it. If you want, you can mention that to get the new tests in the application, obviously the application must be re-built